src/Components/NotificationIntegrationModal/Components/TabComponent.jsx

```
@@ -2,19 +2,21 @@ import React from "react";
-import TextInput from "../../../src/Components/Inputs/TextInput";
-import Checkbox from "../../../src/Components/Inputs/Checkbox";
+import TextInput from "../../../Components/Inputs/TextInput";
+import Checkbox from "../../../Components/Inputs/Checkbox";
 
 const TabComponent = ({ 
   type, 
   integrations, 
   handleIntegrationChange, 
   handleInputChange, 
-  handleTestNotification 
+  handleTestNotification,
```

ESLint

[error] 'handleTestNotification' is missing in props validation

(react/prop-types)